app/views/components/Anchor.scala.html

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  *@
 
+@import config.FrontendAppConfig
+
+@this(appConfig: FrontendAppConfig)
+
 @()(implicit messages: Messages)
 
-    <div class="user-attention">
-        <h2 class="heading-small">@Messages("whatYouNeed.attention.title")</h2>
-        <p>@Messages("whatYouNeed.attention.information")</p>
-    </div>
+<p class="govuk-body">
+ <a href="@{appConfig.amlsFrontendBaseUrl}">@messages("link.return.registration.progress")</a>
```

Comment:
This link should take the user to the registration progress URL. There was a val for this in `FrontendAppConfig`: `registrationProgressUrl`. I put that in my list of unused vals in a comment last time due to the missing link here in the last commit, so I'll take the blame for that, sorry! Can you re-add that val and then update this href.